pages/events/index.js

```diff
@@ -1,19 +1,32 @@
 import Link from "next/link";
 import events from "@/db.json";
+import styled from "styled-components";
 
 function EventsPage() {
   return (
     <>
       <h2>Events</h2>
       <ul>
         {events.map((event) => (
-          <li key={event.id}>
-            <Link href={`/events/${event.id}`}>{event.title}</Link>
-          </li>
+          <StyledCardListContainer key={event.id}>
+            <li key={event.id}>
+              <Link href={`/events/${event.id}`}>{event.title}</Link>
+            </li>
+          </StyledCardListContainer>
```

Comment:
only the wrapping element needs a key

```suggestion
          <StyledCardListContainer key={event.id}>
            <li>
              <Link href={`/events/${event.id}`}>{event.title}</Link>
            </li>
          </StyledCardListContainer>
```